docs/cloud/features/scheduler.md

```diff
@@ -1,3 +1,122 @@
+
+### Concurrent Runs
+
+Unlike the open source built-in scheduler, Tobiko Cloud parallelizes both model executions and run jobs. This means that if one run job is blocked by a long-running model, other independent models can still execute concurrently in separate run jobs.
+
+### Run Pausing
+
+Tobiko Cloud allows you to pause and resume model execution at both the environment and individual model level. This granular control helps manage maintenance windows and troubleshoot issues.
+
+### Isolated Python Environments
+
+Tobiko Cloud automatically manages Python dependencies of your Python macros and models. Each virtual environment has its own isolated Python environment with its own set of dependencies, ensuring that changes in one environment won't affect other environments.
+
+### Improved Concurrency Control
+
+The cloud scheduler ensures that plan applications targeting the same environment are executed sequentially, preventing race conditions and ensuring correct results.
+
+### Access Control
+
+Connection to the data warehouse is managed through Tobiko Cloud. Users can run `run` and `plan` commands without needing to configure database credentials locally.
+
+Additionally, users can control who in their organization can apply changes to which environments and models (coming soon).
+
+## Connection Configuration
+
+To start using the cloud scheduler, you need to configure the connection to your data warehouse first in the Tobiko Cloud UI.
+
+Step 1: Click on the "Settings" tab in the sidebar.
+
+![settings_tab](./scheduler/settings_tab.png)
+
+Step 2: Navigate to the "Connections" tab (1) and click on the "Add Connection" button (2).
+
+![add_connection](./scheduler/add_connection.png)
+
+Step 3: Enter the name of the gateway and the connection configuration in YAML format. The format follows the [connection configuration](../reference/configuration.md#connection) in the SQLMesh reference guide.
+
+**Note:** The gateway name must match the name of the gateway you specify in the `config.yaml` file of your project.
+
+![add_connection_form](./scheduler/add_connection_form.png)
+
+Step 4: Click the "Save" button to add the connection. Upon saving, the connection will be tested and only saved if the connection is successful. The entered configuration is persisted in encrypted form using AES-256 encryption and is only decrypted for execution purposes.
+
+![add_connection_success](./scheduler/add_connection_success.png)
+
+Step 5: Switch to using the cloud scheduler in the project configuration.
+
+**Note:** The gateway name must match the name of the gateway that was specified when adding the connection.
+
+```yaml linenums="1" hl_lines="3 4"
+gateways:
+  my_cloud_gateway:
+    scheduler:
+      type: cloud
+
+default_gateway: my_cloud_gateway
```

Comment:
```suggestion
gateways:
  my_cloud_gateway:
    scheduler:
      type: cloud
    connection: # This connection is used for automatic unit test generation
        type: postgres
        user: postgres
        password: postgres
        host: localhost
        port: 5432
        database: postgres

default_gateway: my_cloud_gateway
```

Reply:
`connection` is not required for this to work. Why would we need this?

Reply:
If we want to enable automatic unit test creation. We debugged this here: https://tobikodata.slack.com/archives/C065Y4BNYJJ/p1734647889738389?thread_ts=1734645873.670879&cid=C065Y4BNYJJ

Reply:
I want to make sure people can still generate unit test configs after they add the tcloud scheduler.

Reply:
most people won’t need that though.

Reply:
yeah, + one of the benefits (`Access Control`) assumes that you don't need the connection config for individual clients. Including this block will make it confusing

Reply:
got it, you can ignore this suggestion then. I'll approve